ios-continuous-integration_5c8a397c2c7d3a154460db00.md

```diff
@@ -1,11 +1,14 @@
-This is a guide to continuous integration for iOS applications created with Xcode.
-Semaphore supports building, testing and deploying Swift, Objective-C and React Native
-projects via [macOS Mojave VM image][macos-mojave] and [a1-standard machine
-types][machine-types].
+This guide gives an overview of CI/CD with Semaphore for apps created with
+Xcode that run on iOS, macOS, watchOS, or tvOS.
+
+Semaphore supports building, testing, and deploying Swift, Objective-C and
+React Native projects. Projects can be built with
+[Xcode 10](macos-mojave-xcode10) or [Xcode 11](macos-mojave-xcode10) running on
+macOS Mojave on an `a1-standard-4` [machine type][machine-types].
 
 **Table of contents**
 
-- [Demo project](#demo-project)
+- [Demo project](#example-project)
```

Comment:
Should be Example project now...

Reply:
Good catch. There were a couple of instances of this mistake. Fixed all.